README.md

```diff
@@ -38,6 +38,8 @@ Finally, compile the client:
 
 The final binary will be at `target/release/mount-s3`.
 
+And for umount use `umount ~/mnt`
```

Comment:
I think we should probably put this down the bottom of this section, right before `### Configuration`?

And we should write out a full sentence: "To unmount the bucket, run `umount ~/mnt`."